plone/scale/tests/test_scale.py

```diff
@@ -102,7 +102,35 @@ def testScaleWithCropDownScale(self):
         self.assertEqual(scaleImage(PNG, 20, 51, "down")[2], (20, 51))
 
     def testNoStretchingDownScale(self):
-        self.assertEqual(scaleImage(PNG, 200, 103, "down")[2], (84, 103))
+        self.assertEqual(scaleImage(PNG, 200, 103, "down")[2], (200, 103))
+
+    def testHugeScale(self):
+        # the image will be cropped, but not scaled
+        self.assertEqual(scaleImage(PNG, 400, 99999, "down")[2], (2, 103))
+
+    def testCropPreWideScaleUnspecifiedHeight(self):
+        image = scaleImage(PNG, 400, None, "down")
+        with open('testCropPreWideScaleUnspecifiedHeight.png', 'wb') as f:
+            f.write(image[0])
```

Comment:
I don't understand why you're writing the result oout to a file. Seems not to be necessary and tests pass without it.

Reply:
Leftovers from debugging.